src/environments/environment-variables.ts

```diff
@@ -47,7 +47,7 @@ const locale: string = env['I18N_LOCALE'] || 'en';
 const sentryDsn: string = env['SENTRY_DSN'];
 const sessionTimeout = Number(env['SESSION_TIMEOUT']) || 30; // minutes
 const pusherCluster = env['PUSHER_CLUSTER'] || 'eu';
-const pusherKey = env['PUSHER_KEY'] || '';
+const pusherKey = env['PUSHER_KEY'] || '22b1d531f1fc10719c7c';
```

Comment:
Do we want to have this in the repository?

Reply:
I will definitely remove it!